docs/source/tutorials/open_images_evaluation.ipynb

```diff
@@ -0,0 +1,444 @@
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Installing FiftyOne\n",
+    "\n",
+    "See [FiftyOne Installation](https://voxel51.com/docs/fiftyone/getting_started/install.html)\n",
+    "for up-to-date instructions."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create a persistent FiftyOne dataset\n",
+    "\n",
+    "The following script loads the data into a FiftyOne [Dataset](https://voxel51.com/docs/fiftyone/api/fiftyone.core.dataset.html#fiftyone.core.dataset.Dataset).\n",
+    "This process copies all labels and metadata to a non-relational database for rapid access and powerful querying, but only paths to the images are stored\n",
+    "in the database, not copies of the images themselves!\n",
+    "\n",
+    "The dataset is set to [persistent](https://voxel51.com/docs/fiftyone/user_guide/using_datasets.html#dataset-persistence)\n",
+    "so that it remains in the database and can be loaded in a new python process."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DATASET_NAME=\"open-images-v4-test\"\n",
+    "IMAGES_DIR=/PATH/TO/IMAGES\n",
+    "BOUNDING_BOXES_EXPANDED=/PATH/TO/test-annotations-bbox_expanded.csv\n",
+    "IMAGE_LABELS_EXPANDED=/PATH/TO/test-annotations-human-imagelabels-boxable_expanded.csv\n",
+    "PREDICTIONS_PATH=/PATH/TO/PREDICTIONS.csv\n",
+    "CLASS_DESCRIPTIONS=/PATH/TO/class-descriptions-boxable.csv\n",
+    "\n",
+    "python open_images_eval/scripts/load_data.py \\\n",
+    "    --bounding_boxes_path ${BOUNDING_BOXES_EXPANDED} \\\n",
+    "    --image_labels_path ${IMAGE_LABELS_EXPANDED} \\\n",
+    "    --predictions_path ${PREDICTIONS_PATH} \\\n",
+    "    --prediction_field_name \"faster_rcnn\" \\\n",
+    "    --class_descriptions_path ${CLASS_DESCRIPTIONS} \\\n",
+    "    --load_images_with_preds \\\n",
+    "    --max_num_images 1000 \\\n",
+    "    ${DATASET_NAME} ${IMAGES_DIR}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### (optional) Visualize the data\n",
+    "\n",
+    "Now that we have a Fiftyone `Dataset`, we can optionally visualize the data before evaluating.\n",
+    "Open up a `python` or `ipython` terminal and run the following:\n",
+    "\n",
+    "```python\n",
+    "import fiftyone as fo\n",
+    "from fiftyone import ViewField as F\n",
+    "\n",
+    "dataset = fo.load_dataset(\"open-images-v4-test\")\n",
+    "\n",
+    "session = fo.launch_app(dataset=dataset)\n",
+    "\n",
+    "# Filter the visible detections by confidence\n",
+    "session.view = dataset.filter_detections(\"faster_rcnn\", F(\"confidence\") > 0.4)\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prepare the ground-truth for evaluation\n",
+    "\n",
+    "Open Images requires \"expanding the hierarchy\" if the ground-truth labels, for\n",
```

Comment:
Missing something

Reply:
?

Reply:
I see what it was now. "if the ground-truth labels" -> "of the ground-truth labels"